knowage-vue/src/i18n/en_US.json

```diff
@@ -127,6 +128,43 @@
             "hint": "The metadata allows you to access and write metadata inside your users document. It's possible to change the type between the allowed ones and to name and describe it. Click on one existing item from the list on the side, or press the \"+\" button to add one.",
             "title": "Metadata Management"
         },
+        "usersManagement": {
+            "title": "Users list",
+            "userId": "User ID",
+            "fullName": "Full Name",
+            "validation": {
+                "sameAsPassword": "Username must be the same as password",
+                "extendedAlphanumeric": "Invalid format",
+                "maximumCharacters": "The maximum length allowed is {length}",
+                "required": "{field} is required"
+            },
+            "detail": "Detail",
+            "roles": "Roles",
+            "attributes": "Attributes",
+            "unlockUser": "Unlock user",
+            "unsavedChangesHeader": "Unsaved changes",
```

Comment:
This label should be common in most list interfaces, move to a common.unsaved section